docs/docs/troubleshooting/LIMITATIONS.md

```diff
@@ -61,6 +61,10 @@ import:
 
 Using this approach will restore the full dataset as part of the Helm install process.
 
+## Crowd
+
+### LoadBalancer service type
+If a kubernetes environment has multiple cluster nodes and `loadBalancer` is used for Crowd `service.type`, Crowd pods may receive client requests from different cluster node ip address. This will cause a session expiry in the client browser. You can resolve it by unticking "Require consistent client IP address" in Session configuration.
```

Comment:
Perhaps we want to say that an alternative way to fix it would be enabling session affinity for the LB, which will result in requests going to the same node:port. Sure thing, when a node is gone the request will go to a different node and this may cause session invalidation